compose-jetsnack/README.md

```diff
@@ -1,54 +1,92 @@
+
+![](screenshots/jetsnack.png)
+
+> To learn more about the Jetsnack application, visit the [README.md of the original project](https://github.com/android/compose-samples/tree/main/Jetsnack).
+
+## Kotlin/Wasm
+
+> **Note**
+> Kotlin/Wasm is Experimental and may be changed at any time. Use it only for evaluation purposes.
+> We would appreciate your feedback on it in the public Slack channel [#webassembly](https://slack-chats.kotlinlang.org/c/webassembly).
+> If you face any issues, please report them on [YouTrack](https://youtrack.jetbrains.com/issue/KT-56492).
 
-# Compose Multiplatform for Web
+Kotlin/Wasm is a new experimental target that enables developers to compile Kotlin code to WebAssembly (Wasm).
+It is designed to offer faster compilation speeds, improved interoperability with JavaScript, and better integration with browsers.
+Wasm's compact and easy-to-parse bytecode potentially leads to faster application startup times, and its statically typed nature can improve application runtime performance.
+
+## Compose Multiplatform for Web
 
 > **Note**
 > Web support is Experimental and may be changed at any time. Use it only for evaluation purposes.
 > We would appreciate your feedback on it in the public Slack channel [#compose-web](https://slack-chats.kotlinlang.org/c/compose-web).
 > If you face any issues, please report them on [GitHub](https://github.com/JetBrains/compose-multiplatform/issues).
 
-You can experiment with sharing your mobile or desktop UIs with the web. Compose for Web is based on [Kotlin/Wasm](https://kotl.in/wasm),
-the newest target for Kotlin Multiplatform projects. It allows Kotlin developers to run their code in the browser with
-all the benefits that WebAssembly has to offer, such as good and predictable performance for your applications.
+Compose for Web is based on [Kotlin/Wasm](https://kotl.in/wasm), the newest target for Kotlin Multiplatform projects.
+It allows you to run your code in the browser with all the benefits that WebAssembly has to offer, such as good and predictable performance for your applications.
 
-# Jetsnack
+## Setup environment
 
-Web version of [Jetsnack application](https://github.com/android/compose-samples/tree/main/Jetsnack) built with Compose Multiplatform
+### IDE
 
-[//]: # (TODO: screenshot ![]&#40;screenshots/jetsnack.png&#41;)
+We recommend using [IntelliJ IDEA 2023.1 or later](https://www.jetbrains.com/idea/) to work with the project.
+It has Kotlin/Wasm support out of the box.
 
-## Build and run
+### Browser
 
-Check out the repo, navigate to the project folder, and use the following commands:
+Almost all modern browsers already support WebAssembly 1.0.
 
-### Run Web version via Gradle
+To set up the environment to run WebAssembly, you only need to enable an experimental [garbage collection feature](https://github.com/WebAssembly/gc):
```

Comment:
it required only for applications built with Kotlin/Wasm

Reply:
with current text it may feel like it's general requirement

Reply:
Sure, updated.